flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/main/java/org/apache/flink/cdc/connectors/base/source/meta/split/SnapshotSplit.java

```diff
@@ -95,6 +95,10 @@ public TableId getTableId() {
         return tableId;
     }
 
+    public static TableId parseTableId(String splitId) {
+        return TableId.parse(splitId.substring(0, splitId.lastIndexOf(":")));
+    }
```

Comment:
This method has been provided by `extractTableId`. Remove this method please.

Reply:
Processed